node/src/Commands.ts

```diff
@@ -744,9 +761,11 @@ export function createZadd(
     let args = [key];
 
     if (options) {
-        if (options.conditionalChange === "onlyIfExists") {
+        if (options.conditionalChange === ConditionalChange.OnlyIfExist) {
             args.push("XX");
```

Comment:
We don't need it anymore with your changes, we can use args.push(options.conditionalSet.valueOf());, no?